src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+    {person.age && (
+      <p className="Person__age">
+        {`I am ${person.age}`}
+      </p>
+    )}
+    <p className="Person__partner">
+      {person.isMarried ? `${person.partnerName} is my ${person.sex === 'f' ? 'husband' : 'wife'}` : 'I am not married'}
```

Comment:
```suggestion
      {person.isMarried 
        ? `${person.partnerName} is my ${person.sex === 'f' ? 'husband': 'wife'}` 
        : 'I am not married'
      }
```
just FYI - it's easier to read if formatted in this way :arrow_up: